modules/df/df_admin/df_admin.info.yml

```diff
@@ -41,6 +41,7 @@ dependencies:
   - import
   - libraries
   - lightning_core
+  - lightning_layout
```

Comment:
Why do we need to do this?  The only thing I can see this does for us is install the lightning_layout too late for config/rewrite to matter, thereby giving us unwanted roles.  Given that df_tools_layout also declares this, it seems this is redundant.